ADR/ADR00TBD-use-rust-for-api.md

```diff
@@ -0,0 +1,39 @@
+# ADR00TBD: Use Rust for our core API
+
+Date: 2022-03-15
+
+## Status
+
+`Proposed`
+
+## Context
+
+> Rust bills itself as _A language empowering everyone to build reliable and efficient software_. It was first released in 2010, 8 months after Go. 
+It is designed to build provablly correct software, with an emphasis on zero runtime failures, and high-quality developer ergonomics.
+
+Our core API is (or will be) a vital piece of government infrastructure. It is important that it is both reliable and fast, without being complicated to develop. Further, the carbon cost of this infrastructure needs to be low.
```

Comment:
How do users access this API? Do they need to use/know Rust, or is it a web service?

Reply:
Apologies, missed this. Web service. The maintenance burden requires organisational knowledge of Rust, which is one of the principle drawbacks in the report.